vendor/bat-native-ledger/src/bat/ledger/internal/publisher/publisher.cc

```diff
@@ -264,21 +263,15 @@ void Publisher::SaveVisitInternal(
   publisher_info->url = visit_data.url;
   publisher_info->status = status;
 
-  bool excluded = IsExcluded(
-      publisher_info->id,
-      server_excluded,
-      static_cast<ledger::PublisherExclude>(publisher_info->excluded));
+  bool excluded =
```

Comment:
The `excluded` field is not included in channel responses and has been removed from `ServerPublisherInfo` structs and tables. It is still available in `PublisherInfo`.